src/timer_esp32.rs

```diff
@@ -40,12 +41,38 @@ fn read_timer_value() -> u64 {
 }
 
 pub fn setup_timer_isr(timg1: TIMG1, clocks: &Clocks) {
-    let mut timer1 = Timer::new(timg1, clocks.apb_clock);
-    interrupt::enable(
-        Cpu::ProCpu,
+    let timg1 = TimerGroup::new(timg1, &clocks);
+    let mut timer1 = timg1.timer0;
+    interrupt::vectored::enable(
```

Comment:
```suggestion
    interrupt::enable(
```

Oops, I think I should have made `vectored` private, as we re-export the public items inside them anyway.